src/Nonlinear/operators.jl

```diff
@@ -339,14 +346,17 @@ end
 function _UnivariateOperator(op::Symbol, f::Function)
     _validate_register_assumptions(f, op, 1)
     f′ = _checked_derivative(f, op)
-    f′′ = _checked_derivative(f′, op)
-    return _UnivariateOperator(f, f′, f′′)
+    return _UnivariateOperator(op, f, f′)
 end
 
 function _UnivariateOperator(op::Symbol, f::Function, f′::Function)
-    _validate_register_assumptions(f′, op, 1)
-    f′′ = _checked_derivative(f′, op)
-    return _UnivariateOperator(f, f′, f′′)
+    try
+        _validate_register_assumptions(f′, op, 1)
+        f′′ = _checked_derivative(f′, op)
+        return _UnivariateOperator(f, f′, f′′)
```

Comment:
I guess one open question is this behavior: do we always try to ForwardDiff `f′`?